composable-architecture-test/src/main/kotlin/composablearchitecture/test/TestStore.kt

```diff
@@ -84,7 +98,9 @@ private constructor(
             fromLocalAction,
             testDispatcher
         )
-
+    
+    @kotlin.ExperimentalStdlibApi
+    @OptIn(kotlin.ExperimentalStdlibApi::class)
```

Comment:
These should not be necessary, we are enabling usage of experimental APIs globally in [`build.gradle.kts`](https://github.com/wearemakery/kotlin-composable-architecture/blob/master/build.gradle.kts#L22).